css/index.css

```diff
@@ -1,3 +1,474 @@
+	padding-right:25px;
+	padding-bottom: 40px;
+
+}
+.bodytop {
+	display: flex;
+	justify-content: space-between;
+}
+.bodytop div {
+	margin-right: 20px;
+	padding: 15px;
+	background-color: lightgray;
+	border: 1px solid black;
+}
+.bodymiddle {
+	display: flex;
+	justify-content: space-between;
+}
+.bodymiddle div {
+	margin-right: 20px;
+	margin-top: 20px;
+	padding: 15px;
+	background-color: lightgray;
+	border: 1px solid black;
+}
+.bodybottom {
+	display: flex;
+	justify-content: space-between;
+}
+.bodybottom div {
+	margin-right: 20px;
+	margin-top: 20px;
+	padding: 15px;
+	background-color: lightgray;
+	border: 1px solid black;
+}
+ button {
+	 padding: 10px;
+	 margin-top: 25px;
+	
+ }
+ ul li{
+	 list-style: square;
+	 list-style-position: inside;
+ }
+ .lastround {
+	 display: flex;
+	 width: 100%;
+	 margin-right: 5%;
+	 margin-left: 5%;
+ }
+ .lastroundleft {
+	width: 50%;
+	margin: 20px;
+	padding: 20px;
+ }
+ .lastroundright {
+	 width: 100%;
+	 margin-right: 5%;
+	 margin-left: 5%;
+	 padding: 5x;
+	 display:flex;
+	 justify-content: center;
+ }
+ .lastroundleft p {
+	 margin-bottom: 20px;
+ }
+ .lastround li {
+	 padding: 5px;
+ }
+ .services-info-img{
+	 justify-content: center;
+
+ }
+
+ /*Media for Tablet */
+ @media screen and (max-width: 800px) {
+    body {
+      background-color:#ffffff;
+    }
+
+    .navbar {
+        display: flex;
+        flex-direction: column-reverse;
+        padding: 20px 0px 40px 0px;
+
+    }
+    
+    .navlinks {
+        width: 100%;
+        display: flex;
+        justify-content: space-between;
+        padding-top: 30px;
+    }
+    .nav a{
+        text-decoration: none;
+        letter-spacing: 3px;
+        color: black;
+  }
+
+  .codepic {
+     display: none;
+  }
+ 
+  .innovation {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+.innovation h1 {
+   text-align: center;
+   padding: 15px;
+   margin-top: 15px;
+
+}
+.companylogo {
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+.innovation h1 {
+    text-align: center;
+    padding: 0px;
+	margin-top: 15px;
+
+}
+
+ 
+}
+  
+  /* Media for Phone */
+  @media screen and (max-width: 500px) {
+    body {
+      background-color:#fff;
+    }
+    .navlinks {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        padding-top: 30px;
+        align-items: center;
+        color: grey;
+    }
+    .navlinks a {
+        border-bottom: 1px solid grey;
+        width: 500px;
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        color: #708090;
+        padding-top: 5%;
+        padding-bottom: 5%;
+    }
+    .top-content {
+        display: flex;
+        flex-direction: column;
+        border-top: 1px solid black;
+        width: 96%;
+        padding: 0px;
+        margin: 0 2%;
+    
+    }
+
+    .navlink1 {
+        border-top: 1px solid grey;
+    }
+    .navlink2 {
```

Comment:
Don't forget to check for empty selectors you can remove. You'll see errors or warnings in your log if there are empty CSS selectors.